src/librustdoc/clean/types.rs

```diff
@@ -1094,6 +1089,7 @@ impl Attributes {
             }
             add_doc_fragment(&mut out, new_frag);
         }
+        out.pop();
```

Comment:
Doesn't `collapsed_doc_value` also need to be updated?

Reply:
I assume you mean [#92078](https://github.com/rust-lang/rust/pull/92078)? They don't conceptually overlap.